<launch>
    <node pkg="rr_platform" type="image_transform" name="image_transform" output="screen">
        <param name="transform_topics" value="/camera/image_rect,/colors_img"/>
    </node>
</launch>